docs/ReleaseNotes.md

```diff
@@ -16,7 +16,7 @@ Another, smaller change that has been made is that by default, new indexes added
 // begin next release
 ### NEXT_RELEASE
 
-* **Bug fix** Fix LiteralKeyExpression hash code [(Issue #1066)](https://github.com/FoundationDB/fdb-record-layer/issues/1066)
+* **Bug fix** Fix 1 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
```

Comment:
I guess there's a slight concern that if there's not a 2.10 release between now and when 3.0 is released, this bug fix will never get a release note. I think we'll probably want to do one last 2.10 release (if there's anything not yet released) before cutting 3.0, though, so probably not a real concern.